cross/cereal/Makefile

```diff
@@ -18,7 +18,8 @@ CMAKE_ARGS += -DWITH_WERROR=no
 
 include ../../mk/spksrc.cross-cmake.mk
 
-ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
-# fix compilation with newer compilers
-CMAKE_ARGS += -DCMAKE_CXX_FLAGS="-D_GLIBCXX_USE_C99 -D_GLIBCXX_USE_C99_MATH" 
+# Fix compilation with newer compilers
+# Flags to be added to CMake toolchain file
+ifeq ($(call version_ge, $(call version_gcc), 7.5),1)
+ADDITIONAL_CXXFLAGS += -D_GLIBCXX_USE_C99 -D_GLIBCXX_USE_C99_MATH
 endif
```

Comment:
Didn't you want to move this to the "CMake toolchain file" in this PR?

Reply:
I thought of that... and I'm not sure about it as it's not "all" cases that needs it...  I'm willing to go as you see fit, worst case we can change it back later on.

Also, if I recall (and I may be wrong) this was used with GCC-7.3 for DSM7-beta.  If that was the case it may not be needed anymore with GCC-7.5 (again, this is a vague souvenir of the gcc version difference between beta and final, I'd need to double check to be sure).